src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/task/BehandlingsstatistikkTask.kt

```diff
@@ -97,7 +101,11 @@ class BehandlingsstatistikkTask(
             relatertEksternBehandlingId = relatertEksternBehandlingId,
             relatertBehandlingId = null,
             behandlingMetode = behandlingMetode,
-            behandlingÅrsak = saksbehandling.årsak
+            behandlingÅrsak = saksbehandling.årsak,
+            kravMottatt = saksbehandling.kravMottatt,
+            årsakRevurdering = årsakRevurdering?.let {
+                ÅrsakRevurderingDto(it.opplysningskilde, it.årsak)
```

Comment:
Burde vi sende kravMottatt og årsakRevurdering uavhengig hvilken hendelse det gjelder (@olekvernberg @charliemidtlyng @eirarset )

Reply:
Ja, jeg tenker det er greit hvis vi har det. Hvis dataen regnes som irrelevant for enkelte hendelsetyper så tror jeg det bare filtreres ut av datavarehus